scene/2d/multimesh_instance_2d.cpp

```diff
@@ -42,8 +42,23 @@
 Callable MultiMeshInstance2D::_navmesh_source_geometry_parsing_callback;
 RID MultiMeshInstance2D::_navmesh_source_geometry_parser;
 
+void MultiMeshInstance2D::_refresh_interpolated() {
+	if (is_inside_tree() && multimesh.is_valid()) {
+		bool interpolated = is_physics_interpolated_and_enabled();
+		multimesh->set_physics_interpolated(interpolated);
+	}
+}
+
+void MultiMeshInstance2D::_physics_interpolated_changed() {
+	_refresh_interpolated();
```

Comment:
Be sure to call the base class `VisualInstance::_physics_interpolated_changed()` here as it isn't done for you.